spec/_layouts/default.html

```diff
@@ -32,76 +31,95 @@
+<body>
+	<div id="segment-header" class="segment">  <!-- segment-header -->
 		<div class="container">  <!-- container -->
-      <a id="logo" no-external="true" href="http://hl7.org/fhir"><img alt="logo fhir" src="{{base}}/dist/fhir-logo-www.png"/> </a>
+      <a id="logo" no-external="true" href="http://hl7.org/fhir"><img alt="logo fhir" src="{{base}}dist/fhir-logo-www.png"/> </a>
 			<div id="hl7-status">
-				<b>FHIRPath STU1 Release</b>
-			</div>       	
-			
+				<b>SMART App Launch Framework</b>
+
+			</div>
+      <div id="hl7-nav">
+         <a id="hl7-logo" no-external="true" href="https://smarthealthit.org/">
+           <img height="50" alt="visit the SMART website" width="42" src="{{base}}dist/smart-logo.svg"/>
+        </a>
+      </div>
+
 			<div id="hl7-nav">
 				 <a id="hl7-logo" no-external="true" href="http://www.hl7.org/">
-           <img height="50" alt="visit the hl7 website" width="42" src="{{base}}/dist/hl7-logo.png"/>
-				</a> 
-			</div>       	
-			<div id="hl7-nav"><a id="hl7-logo" no-external="true" href="http://hl7.org/fhir/search.cfm"><img alt="Search FHIR" src="dist/search.png"/></a></div> 
+           <img height="50" alt="visit the hl7 website" width="42" src="{{base}}dist/hl7-logo.png"/>
+				</a>
+			</div>
+      <!--
+			<div id="hl7-nav"><a id="hl7-logo" no-external="true" href="http://hl7.org/fhir/search.cfm"><img alt="Search FHIR" src="dist/search.png"/></a></div>
+        -->
 		</div>
 		<div class="container">  <!-- container -->
-	</div></div>  <!-- /segment-header -->	
+	</div></div>  <!-- /segment-header -->
 
 	<div id="segment-navbar" class="segment">  <!-- segment-navbar -->
 		<div id="stripe"> </div>
 		<div class="container">  <!-- container -->
    <!-- HEADER CONTENT -->
- 	      		
- 	      		      		
+
+
 			<nav class="navbar navbar-inverse">
 				<div class="container">
 					<button data-target=".navbar-inverse-collapse" class="navbar-toggle" data-toggle="collapse" type="button">
 						<span class="icon-bar"> </span>
 						<span class="icon-bar"> </span>
 						<span class="icon-bar"> </span>
 					</button>
-					<a class="navbar-brand hidden" href="index.html">FHIRPath</a>
+					<a class="navbar-brand hidden" href="#">Navbar</a>
 					<div class="nav-collapse collapse navbar-inverse-collapse">
 						<ul class="nav navbar-nav">
-              <li><a href="index.html">Documentation</a></li>
-              <li><a href="grammar.html">Grammar</a></li>
-              <li><a href="tests.html">Tests</a></li>
-              <li><a href="history.html">Version History</a><!--current--></li>
+              <li><a href="{{base}}index.html">Home</a></li>  <li class="dropdown">
+                  <a href="#" data-toggle="dropdown" class="dropdown-toggle">Conformance<b class="caret">
+                  </b>
+                </a>
+                <ul class="dropdown-menu">
+                  <li><a href="{{base}}conformance/index.html">Declaring Conformance</a></li>
```

Comment:
Is it necessary to put `index.html` here? It'd be nice if the browser path could just be `/conformance` instead of `/conformance.html`. (This is an issue throughout this PR whree `.html` has been added to all links in the spec.)

Reply:
I agree but it generated bad links in the qa output so I added them back in.